README.md

```diff
@@ -8,25 +8,29 @@ The list of providers is published in the form of a statically hosted OPTiMaDe I
 
 If you specifically seek the current list of providers for the latest version of the OPTiMaDe specification, you can access it at this URL:
 
-- [https://www.optimade.org/providers/links/](https://www.optimade.org/providers/links/)
+- [https://www.optimade.org/providers/providers.json](https://www.optimade.org/providers/providers.json)
 
 If you seek the current list of providers for any older version of the OPTiMaDe specification, you can access it using this URL:
 
-- [https://www.optimade.org/providers/*\<version\>*/links/](https://www.optimade.org/providers/<version>/links/)
+- [https://www.optimade.org/providers/*\<version\>*/links](https://www.optimade.org/providers/<version>/links)
```

Comment:
How is `<version>` to be specified? An example would be handy. Will it use [SemVer](https://semver.org/)?

Reply:
`<version>` is a major OPTiMaDe  version. Isn't that clear from the rest of the text, i.e., that this is an OPTiMaDe index meta-database? I suppose it could be clarified.